sig-network/charter.md

```diff
@@ -0,0 +1,46 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
+
+### In scope
+Some notable examples of areas owned by SIG Network:
+- Masquerade, Bridge and Passt binding plugins for the primary network.
+- Bridge, SRIOV and MacVTap bindings for secondary networks.
+- VM networking lifecycle on: creation, migration, hotplug/unplug, restart, stop etc.
+
+#### Code, Binaries and Services
+- [Main repo](https://github.com/kubevirt/kubevirt).
+- Networking related APIs on: VM, VMI, Kubevirt CR, etc.
+- Core network binding infrastructure.
+- Network binding plugin infrastructure.
+- Examples of binding plugin CNIs and sidecars.
+- Virtual Machine/Virtual Machine Instance networking status management.
+- Domain networking configuration.
+- Pod network definition.
+- Unit and e2e tests.
+- Developer and user documentation around networking.
+- Help for onboarding new SIG Network members.
+- Grooming network related bugs.
+
+### Out of scope
+- Node networking, it may be managed by [kubernetes-nmstate](https://github.com/nmstate/kubernetes-nmstate).
+- Host to Pod connectivity configuration, it is managed by external CNIs.
+
+## Roles and Organization Management
+
+This sig adheres to the Roles and Organization Management outlined in [OARP]
```

Comment:
Should OARP have URL link?

Reply:
The url is at the end, now I removed the double OARP tag it works.
It is part of the [sig-charter-template](https://github.com/kubevirt/community/blob/2d3384dafe8a1e1a6f3e780707c8283dbcb8791e/sig-TEMPLATE/sig-charter-template.md#L32-L31).